packages/google_maps_flutter/lib/src/marker.dart

```diff
@@ -216,6 +217,8 @@ class Marker {
   /// Callbacks to receive tap events for markers placed on this map.
   final VoidCallback onTap;
 
+  final ValueChanged<LatLng> onDragEnd;
```

Comment:
Can you add an example to the `examples` directory that illustrates how we can use this feature?

Reply:
Showing a dialog with the new position in the example app.
In our business app we use the drag function to edit map polygons but that's a bit too much work to repeat in the example.